### ISSUE TITLE
Coffeescript tagged template literals are broken

### ISSUE THREAD

**Issue by greghuc** [DanielXMoore/Civet#289]
Hi there

I'm exploring porting a Coffeescript project to Civet, and discovered that CS [tagged template literals](https://coffeescript.org/#tagged-template-literals) are broken.

### Example

Try this in the [Civet Playground](https://civet-web.vercel.app/) - it's the [example used in the CS docs](https://coffeescript.org/#tagged-template-literals). The example breaks at 'upperCaseExpr"""' (note the _no space_ between function call and multiline string). 

```
"civet coffeeCompat"

upperCaseExpr = (textParts, expressions...) ->
  textParts.reduce (text, textPart, i) ->
    text + expressions[i - 1].toUpperCase() + textPart

greet = (name, adjective) ->
  upperCaseExpr"""
               Hi #{name}. You look #{adjective}!
               """

greet("greg", "awesome")
```

### FYI

FYI I wrote the [CS2 support for tagged template literals, and interpolated strings](https://github.com/jashkenas/coffeescript/pull/4352).

Thanks for all your work on Civet. It's looking like a very promising successor to Coffeescript, assuming the migration path is sane.

**Comment by edemaine** [comment]
Nice catch, and thanks for the report! This should be an easy fix.

Note that `foo'''...'''` also doesn't parse (though it's an admittedly less useful form of tagged template literals).

**Comment by greghuc** [comment]
Great thanks!

**Comment by edemaine** [comment]
Should be fixed in the 0.5.58 release!
